src/DataSource/Types.hs

```diff
@@ -847,5 +847,6 @@ renderTicketStatus AnalyzedByScript     = "analyzed-by-script"
 renderTicketStatus AnalyzedByScriptV1_0 = "analyzed-by-script-v1.0"
 renderTicketStatus AnalyzedByScriptV1_1 = "analyzed-by-script-v1.1"
 renderTicketStatus AnalyzedByScriptV1_2 = "analyzed-by-script-v1.2"
+renderTicketStatus ToBeAnalyzed         = "to_be_analysed" -- https://iohk.zendesk.com/agent/admin/tags
```

Comment:
I can't access this link (https://iohk.zendesk.com/agent/admin/tags) so I can't see what's there. Anyhow, I thought `to-be-analyzed` could be instead of `to_be_analysed` to achieve consistency with other tags: they were written in American English and with dashes instead of underscored.

Reply:
The link can be seen by Zendesk admins (currently two of us from TSD).
Yes, I agree, but that's something we had no control over - it was done by the helpdesk team and it's what it is.